distribution/pom.xml

```diff
@@ -218,6 +218,8 @@
                                         <argument>-c</argument>
                                         <argument>org.apache.druid.extensions:druid-protobuf-extensions</argument>
                                         <argument>-c</argument>
+                                        <argument>org.apache.druid.extensions:druid-rocketmq-indexing-service</argument>
```

Comment:
community contributed extensions should be put inside the 'bundle-contrib-ext' profile  (at the bottom of this file)

Reply:
OK! I'll change it.